### ISSUE TITLE
FY22 New Fees & Fee Amounts (On PROD)

### ISSUE THREAD

**Issue by TracyLinder** [cityofaustin/atd-data-tech#6842]
# This single issue covers all ROW folder fee amount updates, and creation of new fees. 

### DS
- [ ] Rename 'Excavation/Concrete Permit Fee' to **'Driveway/Sidewalk Application Fee'**
- [ ] Rename the existing Insert + Run procedure on DS
- [ ] Remove fee from EX Insert + Run
- [ ] Remove from the EX lookup table
- [ ] Update  ⬆ amount from $45 to $195

----

### LC
- [ ] Update fee amount on Concrete License Fee from $50 to $160

----

### TCP Review Fee
- [ ] (Fee code 51283) Update amount from $50 to $220
- [ ] Add these fees for SP, GP

----

### [TURP](https://app.zenhub.com/workspaces/data--technology-services-5caf7dc6ecad11531cc418ef/issues/cityofaustin/atd-data-tech/6564)

- [ ]  (New Fee) Fee Description - 'TURP Permit Fee - New'
Fee amount - $320
GL Account Number - 5125 2400 9100 4087

- [ ] (New Fee) Fee Description - 'TURP Permit Fee - Emergency'
Fee amount - $95
GL Account Number - 5125 2400 9100 4087

- [ ] (New Fee) Fee Description - 'TURP Permit Fee - Extension'
Fee amount - $195
GL Account Number - 5125 2400 9100 4087

- [ ] (Update Amounts) TURP ROW Usage Fees (50150)
![image](https://user-images.githubusercontent.com/49930868/127915489-04bd7189-bc37-4957-a8de-e931553d9594.png)

----
### EX

- [ ]  (New Fee) Fee Description - 'Excavation Permit Fee - New'
Fee amount - $290
GL Account Number - 5125 2400 9100 4057

- [ ]  (New Fee) Fee Description - 'Excavation Permit Fee - Secondary'
Fee amount - $95
GL Account Number - 5125 2400 9100 4057

- [ ]  (New Fee) Fee Description - 'Excavation Permit Fee - Emergency'
Fee amount - $95
GL Account Number - 5125 2400 9100 4057

- [ ]  (New Fee) Fee Description - 'Excavation Permit Fee - Extension'
Fee amount - $175
GL Account Number - 5125 2400 9100 4057

- [ ] (Update Fee Amount) ROW Usage Fee - Street Lanes and ROW Usage Fee - Structure
![image](https://user-images.githubusercontent.com/49930868/127915967-6961fc02-123c-46da-8ffd-c50817346326.png)

<img width="378" alt="Screen Shot 2021-08-06 at 1 23 07 PM" src="https://user-images.githubusercontent.com/49930868/128555324-926f6576-5668-4697-a509-45dd43360de3.png">

**Comment by TracyLinder** [comment]
I don't have the fee codes for a few of these, but this is ready for development. 
FYI @Nadin-Nader @vschandramouli @Sreelatha-BlazeKannan

**Comment by Nadin-Nader** [comment]
**Status:**
Sree is waiting on questions from email to be answered

(_Talk about during ROW Meeting if need be_)

**Comment by TracyLinder** [comment]
**Update**

TURP permits will not be charging a secondary fee. If it is secondary, we should charge the new permit fee. I need to update this issue and #6844

**Comment by TracyLinder** [comment]
AC has been updated; removed TURP Secondary Fee

**Comment by TracyLinder** [comment]
**Status**

- Sree is almost done w/ changes and is doing her testing. 
- Challenge: So many people doing lookup changes; Sree loses her work often. 
- Hoping to be able to send a deployment plan tomorrow.

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder  - Sent DEV Deployment plan of #6842 four reference. Here is the commit ID: 645cc7b66052, please approve and create Task ID.

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder - Resubmitted the commit as per Martin. here are the two commit id's 
c5eed6cb4880 & c162a5b3b23e to create task id's.

**Comment by Nadin-Nader** [comment]
Commit c5eed6cb4880 & c162a5b3b23e  have been sent to the DevOps CM Team for deployment to DEV under **SCTASK0159935**

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan this has been deployed to DEV. Please UAT

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder - Tested the changes in DEV. All looking as expected. TURP fees removed. If approved then this can be deployed to TEST.

**Comment by Sreelatha-BlazeKannan** [comment]
#6842 TEST Deployment plan sent. Kindly approve and promote the changes from DEV to TEST. Here is the commit id’s

c5eed6cb4880 & c162a5b3b23e

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan I have sent this to the DevOps CM Team to deploy to TEST under **SCTASK0163663**

**Comment by Nadin-Nader** [comment]
**Status:**
Waiting on DevOps to deploy to TEST under **SCTASK0163663**

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan @TracyLinder this has been deployed to TEST. Please UAT

**Comment by Nadin-Nader** [comment]
@TracyLinder After Paloma and Kim UATd this on TEST she found that only some of the fees were updated. Come to find out that the commits partially got done. During aour meeting with Martin and George the issue was due to a comma missing and the GL code format changing.
George will send us the new format Sree will fix the comma issue and Martin asked that we recommit the changes  and he will send them through.

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan here is the new Fee Schedule (if needed):

[https://app.zenhub.com/files/140626918/83c3282c-3daa-4a32-a7cf-1a1af978b00b/download](https://app.zenhub.com/files/140626918/83c3282c-3daa-4a32-a7cf-1a1af978b00b/download)

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder - submitted 3rd commit for #6842 which I did as per George. Here is the 3rd commit id for your reference. Please approve and create task id.

71bdb2abbb8e

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder - George pinged me on TEST move of this commit and I tested the changes also; TEST Changes all are working as expected. with all fixes (manual where ever necessary) George provided these two commit numbers for us to use to Promote the changes to PROD in future. 

**d653ce539ea9 - TEST version of LaunchIT branch commit #c5eed6cb4880
**ad0fa52179b9 - TEST version of LaunchIT branch commit #71bdb2abbb8e

While creating task id please use this commit numbers. Ready for PROD. This can be tested by UAT Team.

**Comment by Nadin-Nader** [comment]
## Ready for TEST UAT

**Comment by TracyLinder** [comment]
UAT ISSUE 1
_Reported by Kim_
- [ ] Done/Resolved

**TURP ROW Usage Fees are not updated**
In all tiers the following fees were not correct:

- Sidewalk Space Extension
- Alley extension
- Additional traffic lane extension

**Comment by TracyLinder** [comment]
### UAT ISSUE 2
_Reported by Evelyn_
- [ ] Done/Resolved

**PSM Fees have not been updated w/ new amounts.**

- I did not get the PSM fee updates in the AC, so I will create a separate ticket for that fee update. (It will have to be delivered next week.)

**Comment by TracyLinder** [comment]
### Please move to PROD w/ all existing bugs/issues.

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder - submitted #6842 PROD Deployment plan for your reference. Kindly approve and create task id for me. Since George Moved this card earlier please ask him to promote it to PROD as well. 

**d653ce539ea9 - TEST version of LaunchIT branch commit #c5eed6cb4880
**ad0fa52179b9 - TEST version of LaunchIT branch commit #71bdb2abbb8e

**Comment by Nadin-Nader** [comment]
This has been delivered and has no lagging issues. Closing Card
